ops/terraform/modules/resources/iam/main.tf

```diff
@@ -39,6 +39,7 @@ resource "aws_iam_role_policy" "logs_policy" {
       {
         "Effect": "Allow",
         "Action": [
+          "ec2:DescribeTags",
```

Comment:
If you look below here, the `Resource` scoping on these three feels a bit wide. Very open to suggestions/patches that would narrow it:
1. The `logs:` permissions could be down-scoped to `/bfd/{{ env }}/*`.
2. The `ec2:DescribeTags` permission could be downscoped to just the instance/self.